### PR TITLE
[ja] Update content/ja/docs/concepts/containers/container-lifecycle-hooks.md to the latest.

### LINKED ISSUE #35905: [ja] content/ja/docs/concepts/containers/container-lifecycle-hooks.md is outdated.
**This is a Feature Request**

**What would you like to be added**
We need to update content/ja/docs/concepts/containers/container-lifecycle-hooks.md according to the English text, because it is outdated.

**Why is this needed**
content/ja/docs/concepts/containers/container-lifecycle-hooks.md is outdated.

**Comments**
/language ja
/assign

### PR DESCRIPTION
**Pull request by s-kawamura-w664**
Fixed content/ja/docs/concepts/containers/container-lifecycle-hooks.md according to the English text, because it is outdated.

Fixes #35905

### CODE REVIEW

**Review by t-inu — COMMENTED** [review]
I reviewed and left comments.

**Comment by t-inu** [comment]
https://github.com/kubernetes/website/blob/453736593be0b6a16b4926d6d8d988685984e4ba/content/ja/docs/concepts/containers/container-lifecycle-hooks.md?plain=1#L61
これら２つの実行に -> これら2つの実行に

**Comment by t-inu** [comment]
https://github.com/kubernetes/website/blob/453736593be0b6a16b4926d6d8d988685984e4ba/content/ja/docs/concepts/containers/container-lifecycle-hooks.md?plain=1#L107

Japanese page is existed in this link. The following is better.
/ja/docs/tasks/configure-pod-container/attach-handler-lifecycle-event/

**Comment by s-kawamura-w664** [comment]
@t-inu 
Thank you for your review.
I updated for your 4 comments.
And I checked to jump to Japanese page on netlify preview too.
Could you please confirm my update.

File: content/ja/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -48,13 +47,13 @@ Angularなどのコンポーネントライフサイクルフックを持つ多
...
 コンテナライフサイクル管理フックが呼び出されると、Kubernetes管理システムはフックアクションにしたがってハンドラーを実行します。
-`exec`と`tcpSocket`はコンテナの中で実行され、`httpGet`はkubeletプロセスによって実行されます。
+`httpGet`と`tcpSocket`はkubeletプロセスによって実行され、`exec`はコンテナの中で実行されます。
 
 フックハンドラーの呼び出しは、コンテナを含むPodのコンテキスト内で同期しています。
 これは、`PostStart`フックの場合、コンテナのENTRYPOINTとフックは非同期に起動することを意味します。
 しかし、フックの実行に時間がかかりすぎたりハングしたりすると、コンテナは`running`状態になることができません。
 
-`PreStop`フックはコンテナを停止する信号から非同期で実行されるのではなく、信号が送られる前に実行を完了する必要があります。
+`PreStop`フックはコンテナを停止する信号から非同期で実行されるのではなく、TERMシグナルが送られる前に実行を完了する必要があります。
```
[the remark this answers is not in the document]
Comment (s-kawamura-w664) [reply content/ja/docs/concepts/containers/container-lifecycle-hooks.md]: I fixed as your comment.

File: content/ja/docs/concepts/containers/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -30,8 +30,7 @@ Angularなどのコンポーネントライフサイクルフックを持つ多
 
 `PreStop`
 
-このフックは、APIからの要求、またはliveness probeの失敗、プリエンプション、リソース競合などの管理イベントが原因でコンテナが終了する直前に呼び出されます。コンテナがすでに終了状態または完了状態にある場合、preStopフックの呼び出しは失敗します。
-これはブロッキング、つまり同期的であるため、コンテナを停止する信号が送信される前に完了する必要があります。
+このフックは、APIからの要求、またはliveness/startup probeの失敗、プリエンプション、リソース競合などの管理イベントが原因でコンテナが終了する直前に呼び出されます。コンテナがすでに終了状態または完了状態にある場合には`PreStop`フックの呼び出しは失敗し、コンテナを停止するTERMシグナルが送信される前にフックは完了する必要があります。`PreStop`フックが実行される前にPodの終了猶予期間のカウントダウンが開始されるので、ハンドラの結果に関わらず、コンテナはPodの終了猶予期間内に最終的に終了します。
```
[the remark this answers is not in the document]
Comment (s-kawamura-w664) [reply content/ja/docs/concepts/containers/container-lifecycle-hooks.md]: I fixed as your comment.

**Review by t-inu — COMMENTED** [review]
/lgtm

**Pull request merged**